account_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py

```diff
@@ -2,59 +2,48 @@
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0
 
 from odoo import fields
-from odoo.tests import TransactionCase
+from odoo.tests import Form, TransactionCase
 
 
 class TestAccountBankStatementImportMoveLine(TransactionCase):
     @classmethod
     def setUpClass(cls):
-        super(TestAccountBankStatementImportMoveLine, cls).setUpClass()
-
-        cls.account_type = cls.env["account.account.type"].create(
-            {"name": "Test Account Type", "type": "other", "internal_group": "asset"}
-        )
-
-        cls.a_receivable = cls.env["account.account"].create(
+        super().setUpClass()
+        cls.env.user.groups_id = [(4, cls.env.ref("account.group_account_user").id)]
+        cls.a_income = cls.env["account.account"].create(
```

Comment:
Don't create accounts and just convert the test to post_install. Then, you should create the invoice without a form, as it should work without issues. Check for example this method that creates invoices without using Form: https://github.com/OCA/commission/blob/d9e3e8e944e2fd2ec8eba6e0efea4ddb01e9eb5b/account_commission/models/commission_settlement.py#L129